web/src/beta/features/Editor/Map/DataSourceLayerCreator/KML/hooks.tsx

```
@@ -0,0 +1,73 @@
+    ],
+    [t]
+  );
+
+  const handleSubmit = () => {
+    let parsedValue = undefined;
+
+    if (sourceType === "value" && value !== "") {
+      {
+        parsedValue =
+          "data:text/plain;charset=UTF-8," + encodeURIComponent(value);
+      }
```

Biome

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)